### PR TITLE
feat: add module for component homepage and routing

### PR DESCRIPTION
**Pull request by Darya2311**

### CODE REVIEW

**Review by yarisleiv — APPROVED** [review]
Well done!

**Review by RuslanTanasiichuk — APPROVED** [review]

File: src/app/homepage/homepage-routing.module.ts [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import { NgModule } from '@angular/core';
+import { RouterModule, Routes } from '@angular/router';
+
+const routes: Routes = [];
+
+@NgModule({
+  imports: [RouterModule.forRoot(routes)],
```
Comment (ViktorRiabinin) [inline src/app/homepage/homepage-routing.module.ts:7]: This is not a root config, 'forChild' must be used here

File: src/app/homepage/homepage.module.ts [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  ],
+  imports: [
+    BrowserModule,
+    HomepageRoutingModule
+  ],
+  providers: [],
+  exports: [
+    HomepageComponent
+  ],
+  bootstrap: [HomepageComponent]
```
Comment (ViktorRiabinin) [inline src/app/homepage/homepage.module.ts:19]: 'bootstrap' property is not needed here

File: src/app/homepage/homepage.module.ts [hunk 3]
```diff
@@ -0,0 +1,21 @@
+import { NgModule } from '@angular/core';
+import { BrowserModule } from '@angular/platform-browser';
+
+import { HomepageComponent } from './homepage.component';
+import {HomepageRoutingModule} from "./homepage-routing.module";
```
Comment (ViktorRiabinin) [inline src/app/homepage/homepage.module.ts:5]: If you run tslint it shows an error here. Please configure your IDE to show lint errors

**Review by ViktorRiabinin — APPROVED** [review]

**Pull request merged**